src/lib/queue.ts

```diff
@@ -68,16 +94,25 @@ export class Queue<K extends KubernetesObject> {
       this.#pendingPromise = true;
 
       // Reconcile the element
-      if (this.#reconcile) {
-        Log.debug(`Reconciling ${element.item.metadata!.name}`);
-        await this.#reconcile(element.item, element.type);
-      }
+      const note = {
+        queue: this.label(),
+        item: {
+          name: element.item.metadata?.name,
+          namespace: element.item.metadata?.namespace,
+          resourceVersion: element.item.metadata?.resourceVersion,
+        },
+      };
+      Log.info(note, "Reconciling");
```

Comment:
```suggestion
      Log.debug(note, "Reconciling");
```

Reply:
This is because if they are reconciling against secrets, they will be exposed (with b64, of course) in controller logs and there will be no way of turning them off